src/Tests/TestProfiler/EnvProfiler/EnvProfiler.cpp

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "EnvProfiler.h"
+#include "tstring.h"
+#include "tostream.h"
+#include <iostream>
+#include <iomanip>
+
+#define HRESULTHEX(x) \
+   "0x" << std::setw(8) << std::setfill('0') << std::hex << x
+
+GUID EnvProfiler::GetClsid()
+{
+    // {6A494330-5848-4A23-9D87-0E57BBF6DE79}
+    return { 0x6A494330, 0x5848, 0x4A23,{ 0x9D, 0x87, 0x0E, 0x57, 0xBB, 0xF6, 0xDE, 0x79 } };
+}
+
+STDMETHODIMP EnvProfiler::Initialize(IUnknown *pICorProfilerInfoUnk)
+{
+    HRESULT hr = S_OK;
+
+    if (FAILED(hr = ProfilerBase::Initialize(pICorProfilerInfoUnk)))
+    {
+        std::cerr << "Failed to initialize profiler." << std::endl;
```

Comment:
Consider a level of indirection for logging. I would not be surprised if we changed the sink at some point.

Reply:
Absolutely. But I don't want to invest in a logging infrastructure yet in this PR.